### PR TITLE
fix(feature media): Fix publish when feature media and/or media descr…

### PR DESCRIPTION
**Pull request by ioanpocol**
…iption are required

### CODE REVIEW

File: apps/validate/validate.py [hunk 1]
```diff
@@ -159,6 +159,17 @@ def _sanitize_fields(self, doc, validator):
...
+    def _process_media(self, doc):
+        """If feature media is required it should be present for validation on doc not only on associations
+        If media_description is required it should be present for validation on doc not only on
+        associations->featuremedia
+        :param doc: Article to be validated
+        """
+        if 'associations' in doc and 'featuremedia' in doc['associations']:
+            doc['feature_media'] = doc['associations']['featuremedia']
+            if 'description_text' in doc['associations']['featuremedia']:
+                doc['media_description'] = doc['associations']['featuremedia']['description_text']
```
Comment (petrjasek) [inline apps/validate/validate.py]: shouldn't it be stored there then?

Comment (ioanpocol) [inline apps/validate/validate.py]: I think no, this is just for validation and to keep the validation simple. There is a well defined logic around associations so I don't know if we have to change it.

Comment (petrjasek) [inline apps/validate/validate.py]: ok

**Comment by ioanpocol** [comment]
done

**Pull request merged**